File: 2-semestre/uri-online-judge/javascript/1015.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let linha1 = lines.shift().split(" ");
let x1 = parseFloat(linha1[0]);
let y1 = parseFloat(linha1[1]);

let linha2 = lines.shift().split(" ");
let x2 = parseFloat(linha2[0]);
let y2 = parseFloat(linha2[1]);

let distancia = Math.sqrt((x2 - x1) * (x2 - x1) + (y2 - y1) * (y2 - y1));

console.log(distancia.toFixed(4));

File: 2-semestre/uri-online-judge/c/1015.c
#include <stdio.h>
#include <math.h>

int main() {
    double x1, y1, x2, y2, distancia;
    
    scanf("%lf", &x1);
    scanf("%lf", &y1);
    scanf("%lf", &x2);
    scanf("%lf", &y2);
    distancia = sqrt((x2 - x1) * (x2 - x1) + (y2 - y1) * (y2 - y1));
    
    printf("%.4lf\n", distancia);
    
    return 0;
}
